applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.h

```diff
@@ -155,40 +165,72 @@ class InterfacePreprocessCondition
     ///@{
 
     /**
-     * Creates a new condition with a giving name
+     * @brief Creates a new properties (contaning just values related with contact)
+     * @details These values are removed from the original property (in order to reduce overload of properties on the original elements)
+     * @param rOriginPart The original model part
+     * @return A map containing new properties
+     */
+
+    std::unordered_map<IndexType, Properties::Pointer> CreateNewProperties(ModelPart& rOriginPart);
+
+    /**
+     * @brief Copies a value from the original property to the new one
+     * @param pOriginalProperty The original property
+     * @param pNewProperty The new property
+     * @param rVariable The variable to copy an erase
+     */
+    template<class TClass>
+    void CopyProperties(
+        Properties::Pointer pOriginalProperty,
+        Properties::Pointer pNewProperty,
+        const Variable<TClass>& rVariable,
+        const bool AssignZero = true
+        )
+    {
+        if(pOriginalProperty->Has(rVariable)) {
+            const TClass& value = pOriginalProperty->GetValue(rVariable);
+            pNewProperty->SetValue(rVariable, value);
+        } else if (AssignZero) {
+            KRATOS_INFO("InterfacePreprocessCondition") << "WARNING:: Property " << rVariable.Name() << " not available. Assigning zero value" << std::endl;
```

Comment:
`KRATOS_INFO` and then Warning? => Maybe `KRATOS_WARNING`?

Reply:
I will remove the WARNING, the thing is that I like to run the tests having only dots, and the tests show the warnings